### ISSUE TITLE
Add a "conformanceMapping" property to rules to better map to ACT requirements

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3459]
A challenge in implementing ACT rules has been that which WCAG success criteria fail isn't always the same as what success criteria you're testing. For example if you want to test SC 1.4.3 Contrast (Minimum), any failure of that is also a failure of SC 1.4.6 Contrast (Enhanced). Translated to axe-core, that would suggest giving the `color-contrast` rule the `wcag146` tag. The problem there though, is if you're actually trying to test SC 1.4.6, you need to run the `color-contrast-enhanced` rule, not `color-contrast` which is much stricter. This issue exists on several rules. 

To be able to accurately report which success criteria a rule fails, I think we need 
Because ACT now requires tools accurately report which success criteria are failed for a particular error, I think we need to record which success criteria fail, separately of which tags a rule has. I'd like to use namespaced URIs for this, which are then easy to convert to EARL. This would look something like this:

```json
{
  "id": "color-contrast",
  "tags": ["cat.color", "wcag2aa", "wcag143"],
  "conformanceMapping": [
    "WCAG2:contrast-minimum",
    "WCAG2:contrast-enhanced"
  ]
}
```

These IDs correspond to the IDs used in WCAG 2 itself. Doing this also lets us map explicitly to requirements in WAI-ARIA. We'd use IDs of paragraphs that contain the requirement, similar to how ACT rules have done this.

The `conformanceMapping` property does not need to be reported in the results. It does need to be returned from `axe.getRules()`.

The following rules need additional mappings that are different from how they are tagged:

- [ ] link-name: 2.4.9 Link Purpose (Link Only)
- [ ] area-alt: 2.4.9 Link Purpose (Link Only)
- [ ] scrollable-region-focusable: 2.1.3 Keyboard (No Exception)
- [ ] color-contrast: 1.4.6 Contrast (Enhanced)

**Comment by WilcoFiers** [comment]
We're not doing this. This isn't going to be necessary.
